scripts/__init__.py

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env python
```

Comment:
This line should not be necessary, as this file is not executable.

Reply:
Sorry, I had a couple of commits from another branch since I did not branch off master. removed those commits which had this file. The file is not part of this commit